src/devices/BrickPi3/BrickPi3.csproj

```diff
@@ -0,0 +1,40 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <EnableDefaultItems>false</EnableDefaultItems>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Device.Gpio" Version="0.1.0-prerelease.19078.2" />
+    <None Include="README.md" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="BrickPi3.cs" />
+    <Compile Include="SPIExceptions.cs" />
+    <Folder Include="Models\" />
+    <Compile Include="Models\BrickPiInfo.cs" />
+    <Compile Include="Models\BrickPiVoltage.cs" />
+    <Compile Include="Models\Motors.cs" />
+    <Compile Include="Models\Sensors.cs" />
```

Comment:
NIT: In order to avoid all the Compile lines, one thing you could do is globbing, like:

```xml
<ItemGroup>
  <Compile Include="Extensions/*.cs" />
  <Compile Include="Models/*.cs" />
  <Compile Include="Movement/*.cs" />
  <Compile Include="Sensors/*.cs" />
  <Compile Include="BrickPi3.cs" />
</ItemGroup>
```

I think that looks a bit cleaner.

Reply:
yes, much cleaner, integrated